docs/configuration.md

```diff
@@ -0,0 +1,7 @@
+# Custom configuration
+
+
+
+The default logging path is `$HOME` To change that, add `set -g @logging-path "path"` to `.tmux.conf` file 
+
+
```

Comment:
Can we just cleanup the unnecessary lines around the text?
From what I can see it does not impact how markdown renders https://github.com/tmux-plugins/tmux-logging/blob/50427989f41dc5a50bfbb80dde0661d997dc0bc7/docs/configuration.md